app/presenters/work_presenter.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+# Presents a work
+class WorkPresenter < ApplicationPresenter
+  def initialize(work:, work_form:)
+    @work = work
+    @work_form = work_form
+    super()
+  end
+
+  attr_reader :work, :work_form
+
+  delegate :status_message, :editable?, to: :status_presenter
+  delegate :collection, :created_at, :druid, :user, to: :work
+  delegate :abstract, :contact_emails_attributes, :keywords_attributes, :license, :other_work_subtype,
```

Comment:
Do we want to expose the `*_attributes` outside the presenter?

Reply:
I don't think so. Most of this can be private.

Reply:
Done.